projects/ngrx.io/content/guide/store/testing.md

```diff
@@ -58,6 +58,24 @@ describe('Auth Guard', () => {
 });
 </code-example>
 
+### Using Mock Selectors
+
+`MockStore` also provides the ability to mock individual selectors to return a passed value using the `overrideSelector()` method. When the selector is invoked by the `select` method, the returned value is overridden by the passed value, regardless of any state snapshot provided in `provideMockStore()`. 
+
+`overrideSelector()` returns a `MemoizedSelector`. To update the mock selector to return a different value, use the `MemoizedSelector`'s `setResult()` method.
+
+`overrideSelector()` supports mocking the `select` method (used in RxJS pipe) and the `Store` `select` instance method using a string or selector.
```

Comment:
I think you wanted to say `setResult` here?
Could we also simplify the wording as the following?

```suggestion
`setResult()` supports mocking the `select` method and returns the given input provided in `setResult()`.
```

Reply:
I intended to summarize the note from #1688:

> Covers mocking different types of selectors:
> 
> ```
> store.select('slice');
> ```
> 
> ```
> const selector = createSelector(state, state => state.slice);
> store.select(selector);
> ```
> 
> ```
> const selector = createSelector(state, state => state.slice);
> store.pipe(select(selector));
> ```